Costume/228_Houndour.yaml

```diff
@@ -0,0 +1,21 @@
+species: Houndour
+imagename: Nickit
+displayname: Nickit
+setname: Costume
+tags: [costume]
+item: [Black Glasses]
+ability: [Unburden]
+moves:
+    - [Night Slash]
+    - [Sucker Punch]
+    - [Quick Attack]
+    - [Nasty Plot]
```

Comment:
nasty plot and 3 physical attacks does not work

Reply:
I get why they added it in, just to match the character. Either a Dark-type special attack or an attack-boosting move would work here.

Reply:
Looking at Nickit closer. Apparently they don't have ANY special attacks they can learn besides Swift and Mud Shot, which theoretically could work, but practically don't fit this set.
Alternatively I could figure out something else to replace Nasty Plot, which I assume is probably the better option here.